.github/actions/setup-sccache/action.yml

```diff
@@ -0,0 +1,69 @@
+name: "Setup sccache"
+description: "Sets up sccache with S3 or local disk configuration"
+
+inputs:
+  disk-max-size:
+    description: "The maximum size of the local disk cache in MB if S3 is unavailable."
+    required: true
+  disk-cache-key:
+    description: "The key to use for the local disk cache."
+    required: true
+  s3-bucket:
+    description: "The s3 bucket to use for cache storage."
+    required: false
+    default: "toolchain-sccache-swift-build"
```

Comment:
Can we leave this unset so that someone else using this functionality configures it according to their environment?

Reply:
Sure